### PR TITLE
feat(pci.private-network): Add link to string 'En savoir plus ici' on vlan config step

### PR DESCRIPTION
**Pull request by Tsiorifamonjena**
ref: TAPC-735

| Question         | Answer
| ---------------- | ---
| Branch?          | `develop` 
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #TAPC-735
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

### CODE REVIEW

**Review by fredericvilcot — APPROVED** [review]

**Review by SimonChaumet — APPROVED** [review]
I'm a bit worried about rerendering on the trans component, can you check that there isn't too much rerenders ?

**Comment by Tsiorifamonjena** [comment]
> I'm a bit worried about rerendering on the trans component, can you check that there isn't too much rerenders ?

@SimonChaumet that's seem good to me but I'm interested in your point of view. By the way a rework will coming soon for the entire page then I'll make sure we optimize

const

File: packages/manager/apps/pci-private-network/src/pages/new/steps/ConfigurationStep.tsx [hunk 1]
```diff
@@ -113,6 +113,10 @@ export default function ConfigurationStep({
     GUIDE_LINKS.REGION_AVAILABILITY[ovhSubsidiary] ||
     GUIDE_LINKS.REGION_AVAILABILITY.DEFAULT;
 
+  // TODO: verify rerender
```
Comment (ThibaudCrespin) [inline packages/manager/apps/pci-private-network/src/pages/new/steps/ConfigurationStep.tsx:116]: something you forgot or legit to keep this TODO here ?

File: packages/manager/apps/pci-private-network/src/pages/new/steps/ConfigurationStep.tsx [hunk 2]
```diff
@@ -390,7 +393,21 @@ export default function ConfigurationStep({
...
+                          <a
+                            href={VLAN_GUIDE_URL}
+                            target="_blank"
+                            rel="noreferrer"
+                          >
+                            {tCommon('common_find_out_more_here')}
+                          </a>
+                        ),
+                      }}
+                    />
```
Comment (tibs245) [inline packages/manager/apps/pci-private-network/src/pages/new/steps/ConfigurationStep.tsx:410]: I don't want block this PR but why don't use a simple : 

```suggestion
                    {t('pci_projects_project_network_private_create_vlan_tip")}
                    <a
                      href={VLAN_GUIDE_URL}
                      target="_blank"
                      rel="noreferrer"
                    >
                      {tCommon('common_find_out_more_here')}
                    </a>
```

?

Comment (tibs245) [reply packages/manager/apps/pci-private-network/src/pages/new/steps/ConfigurationStep.tsx:410]: And replace `<a />` by Link component for have unified behaviour 🙂  
`packages/manager-react-components/src/components/typography/links/links.component.tsx`

https://ovh.github.io/manager/storybook-static/index.html?path=/story/typography-links--external-link

Comment (Tsiorifamonjena) [reply packages/manager/apps/pci-private-network/src/pages/new/steps/ConfigurationStep.tsx]: You are right, nice catch thank you

**Review by antonymarion — APPROVED** [review]

**Review by Eric-ciccotti — APPROVED** [review]

**Review by SimonChaumet — APPROVED** [review]

**Review by fredericvilcot — APPROVED** [review]

**Review by SimonChaumet — APPROVED** [review]

**Review by fredericvilcot — APPROVED** [review]

**Review by tibs245 — APPROVED** [review]

**Pull request merged**